rules/react/walmart.js

```diff
@@ -19,16 +19,28 @@ module.exports = {
     "react/jsx-closing-bracket-location": [2, 'tag-aligned'],
     // Enforce or disallow spaces inside of curly braces in JSX attributes
     "react/jsx-curly-spacing": 0,
+    // Enforce event handler naming conventions in JSX
+    "react/jsx-handler-names": 0,
     // Validate props indentation in JSX
     "react/jsx-indent-props": [2, 2],
+    // Validate JSX has key prop when in array or iterator
+    "react/jsx-key": 0,
```

Comment:
React developer warnings are already going to complain about this. Might as well have the linter catch it early
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md